src/core/overlay/position/global-position-strategy.ts

```diff
@@ -96,12 +84,16 @@ export class GlobalPositionStrategy implements PositionStrategy {
     element.style.bottom = this._bottom;
     element.style.right = this._right;
 
-    // TODO(jelbourn): we don't want to always overwrite the transform property here,
-    // because it will need to be used for animations.
-    let tranlateX = this._reduceTranslateValues('translateX', this._translateX);
-    let translateY = this._reduceTranslateValues('translateY', this._translateY);
+    // Wait one frame for the content to be updated so that we can calculate the correct offset
+    requestAnimationFrame(() => {
+      // TODO(jelbourn): we don't want to always overwrite the transform property here,
+      // because it will need to be used for animations.
+
+      let translateX = Math.round(element.offsetWidth * -0.5);
+      let translateY = Math.round(element.offsetHeight * -0.5);
```

Comment:
It looks like you're _always_ centering the element now? It should only do that when it's configured to be centered.